### PR TITLE
[#56] Change application name and package name

### PR DESCRIPTION
**Pull request by chornerman**
#56

## What happened 👀

Change application name and package name from the default one to:

**Staging**
- App name: `Survey Staging`
- Package name: `co.nimblehq.chorn.survey.staging`

**Production**
- App name: `Survey`
- Package name: `co.nimblehq.chorn.survey`

## Insight 📝

N/A

## Proof Of Work 📹

App run normally

### CODE REVIEW

**Review by lydiasama — APPROVED** [review]
@chornerman I'm curious, do you use the script when generating the project? 🤔  Why do we need to change the package name manually? 😄

**Comment by chornerman** [comment]
> @chornerman I'm curious, do you use the script when generating the project? 🤔 Why do we need to change the package name manually? 😄

@lydiasama I'm curious too, thought it was already set up for me 🙈 Not sure what I did wrong.

File: app/src/main/AndroidManifest.xml [hunk 1]
```diff
@@ -1,11 +1,11 @@
 <?xml version="1.0" encoding="utf-8"?>
 <manifest xmlns:android="http://schemas.android.com/apk/res/android"
-    package="co.nimblehq.template.compose">
+    package="co.nimblehq.chorn.survey">
 
     <uses-permission android:name="android.permission.INTERNET" />
 
     <application
-        android:name=".MainApplication"
+        android:name="co.nimblehq.chorn.survey.MainApplication"
```
Comment (hoangnguyen92dn) [inline app/src/main/AndroidManifest.xml:8]: @chornerman Can we revert this back? The change is unnecessary here.

Comment (chornerman) [reply app/src/main/AndroidManifest.xml]: @hoangnguyen92dn NIce catch 😄 Updated

**Review by hoangnguyen92dn — APPROVED** [review]

**Pull request merged**